### PR TITLE
 Add checker irreparable test

### PR DESCRIPTION
**Pull request by mniewrzal**

### CODE REVIEW

File: satellite/satellitedb/irreparabledb.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ func (db *irreparableDB) IncrementRepairAttempts(ctx context.Context, segmentInf
 		dbxInfo.RepairAttemptCount++
 		updateFields := dbx.Irreparabledb_Update_Fields{}
 		updateFields.RepairAttemptCount = dbx.Irreparabledb_RepairAttemptCount(dbxInfo.RepairAttemptCount)
+		updateFields.SegDamagedUnixSec = dbx.Irreparabledb_SegDamagedUnixSec(segmentInfo.RepairUnixSec)
```
Comment (kaloyan-raev) [inline satellite/satellitedb/irreparabledb.go]: How does this change relate to the test case?

File: pkg/datarepair/checker/checker_test.go [hunk 2]
```diff
@@ -104,3 +105,77 @@ func TestOfflineNodes(t *testing.T) {
...
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		time.Sleep(2 * time.Second)
+
+		const numberOfNodes = 10
+		pieces := make([]*pb.RemotePiece, 0, numberOfNodes)
+		// use online nodes
+		for i, storagenode := range planet.StorageNodes {
+			pieces = append(pieces, &pb.RemotePiece{
+				PieceNum: int32(i),
+				NodeId:   storagenode.Identity.ID,
```
Comment (kaloyan-raev) [inline pkg/datarepair/checker/checker_test.go]: You can use the shorter `storagenode.ID()`.

File: satellite/satellitedb/irreparabledb.go (hunk 1 above)
Comment (mniewrzal) [reply satellite/satellitedb/irreparabledb.go]: I asked @aligeti and he said that repair attempt time should always be updated with attempt count. Turns out that it wasn't updated.

File: pkg/datarepair/checker/checker_test.go [hunk 3]
```diff
@@ -104,3 +105,77 @@ func TestOfflineNodes(t *testing.T) {
...
+			},
+		}
+
+		// put test pointer to db
+		pointerdb := planet.Satellites[0].Metainfo.Service
+		err := pointerdb.Put(pointer.Remote.PieceId, pointer)
+		assert.NoError(t, err)
+
+		checker := planet.Satellites[0].Repair.Checker
+		err = checker.IdentifyInjuredSegments(ctx)
```
Comment (egonelbre) [inline pkg/datarepair/checker/checker_test.go]: Note directly invoking those service methods may cause races in the future.

Comment (mniewrzal) [reply pkg/datarepair/checker/checker_test.go]: Yes, I know but I think we don't have easy way to disable services triggered by timers. I will add comment about it.

**Pull request merged**